src/lib/ops_and_meta_predicates.pl

```diff
@@ -5,9 +5,16 @@
 
 :- op(1199, fx, meta_predicate).
 
-/* this is an implementation specific declarative operator used to implement call_with_inference_limit/3
-   and setup_call_cleanup/3. switches to the default trust_me and retry_me_else. Indexing choice
-   instructions are unchanged. */
+% Implementation specific declarative operator used to implement
```

Comment:
@triska I've re-added "implementation specific" part. Merged code imho didn't change meaning of the original comment.